materialize-elasticsearch/schemabuilder/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "schema-builder"
+version = "0.1.0"
+authors = ["Jixiang Jiang <jixiang@estuary.dev>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+json="*"
+serde = "~1"
+serde_json = "~1"
+thiserror = "*"
+url = "*"
+tracing = "*"
+tracing-subscriber = {version = "*", features = ["time", "json", "env-filter"]}
+
+doc = { git = "https://github.com/estuary/flow", version = "0.0.0"}
```

Comment:
Aha.. I remembered I Googled and found [this](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html). Interestingly, I started with the local path approach first, which has a `version` spec, and then transferred back to `git`, and checked the version is effective for git as well.

When thinking on this again, I feel the git rev, which allows us to pin down a particular version of Flow Repo is more helpful here - since right now the changes in Flow repo are unaware of the connectors. Connector needs to find the correct version of flow crates and test/release them.. (I am open to any other suggestions on this as well.)

Reply:
I don't think the `rev` is necessary, since we check in the `Cargo.lock` file. [Here's the docs on that](https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html) if you're curious. But the TLDR is that Cargo.lock has all the resolved versions of all your dependencies, so they'll only be updated when you explicitly run `cargo update`, at which point it'll pull the lastest master branch.

Reply:
Done.